src/System.Private.CoreLib/Resources/Strings.resx

```diff
@@ -1999,36 +1954,12 @@
   <data name="CancellationToken_CreateLinkedToken_TokensIsEmpty" xml:space="preserve">
     <value>No tokens were supplied.</value>
   </data>
-  <data name="CancellationToken_SourceDisposed" xml:space="preserve">
-    <value>The CancellationTokenSource associated with this CancellationToken has been disposed.</value>
-  </data>
   <data name="CancellationTokenSource_Disposed" xml:space="preserve">
     <value>The CancellationTokenSource has been disposed.</value>
   </data>
   <data name="ConcurrentCollection_SyncRoot_NotSupported" xml:space="preserve">
     <value>The SyncRoot property may not be used for the synchronization of concurrent collections.</value>
   </data>
-  <data name="event_SpinLock_FastPathFailed" xml:space="preserve">
-    <value>SpinLock beginning to spin.</value>
-  </data>
-  <data name="event_SpinWait_NextSpinWillYield" xml:space="preserve">
-    <value>Next spin will yield.</value>
-  </data>
```

Comment:
Looks like it isn't used too. Am I right?

Reply:
I believe you're correct.